test/WebSites/FiltersWebSite/Controllers/ActionFilterController.cs

```diff
@@ -37,6 +37,7 @@ public override void OnActionExecuting(ActionExecutingContext context)
             {
                 filters = (List<ContentResult>)obj;
             }
+            else
```

Comment:
have no idea how the test was passing.  this is another separate commit to keep the quick fix obvious.